# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d9/SwapChain9.cpp
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d9/SwapChain9.cpp
--- a/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d9/SwapChain9.cpp	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d9/SwapChain9.cpp	Sat Sep 07 13:38:36 2019 +0000
@@ -360,7 +360,7 @@
     // windows is in the process of entering/exiting fullscreen. This code doesn't seem to have any
     // documentation.  The device appears to be ok after emitting this error so simply return a
     // failure to swap.
-    if (result == static_cast<HRESULT>(0x88760873))
+    if (result == static_cast<HRESULT>(0x88760873) || result == static_cast<HRESULT>(0x88760872))
     {
         return EGL_BAD_MATCH;
     }